Detectors/PHOS/calib/src/PHOSBadMapCalibDevice.cxx

```diff
@@ -253,26 +253,26 @@ void PHOSCalibCollector::writeOutputs()
   fHistoOut.Close();
 }
 
-void PHOSCalibCollector::fillTimeMassHisto(const FullCluster& clu)
+void PHOSCalibCollector::fillTimeMassHisto(const Cluster& clu)
 {
-  // Fill time distributions only for cells in cluster
-  if (mMode == 0) {
-    auto cluList = clu.getElementList();
-    for (auto ce = cluList->begin(); ce != cluList->end(); ce++) {
-      short absId = ce->absId;
-      if (ce->isHG) {
-        if (ce->energy > mEminHGTime) {
-          mHistos[kTimeHGPerCell].Fill(absId, ce->time);
-        }
-        mHistos[kTimeHGSlewing].Fill(ce->time, ce->energy);
-      } else {
-        if (ce->energy > mEminLGTime) {
-          mHistos[kTimeLGPerCell].Fill(absId, ce->time);
-        }
-        mHistos[kTimeLGSlewing].Fill(ce->time, ce->energy);
-      }
-    }
-  }
+  // // Fill time distributions only for cells in cluster
```

Comment:
if commented code is obsolete, please remove it

Reply:
This class still should be implemented. This is seed for final class